src/Template/Element/form.ctp

```diff
@@ -8,6 +8,12 @@
     <div class="row">
         <div class="col-lg-<?= $this->exists('form.sidebar') ? '8' : '12' ?>">
             <?= $this->Form->inputs($fields, ['legend' => false]); ?>
+            <?php
+            $this->Html->scriptBlock(
+                '$("select[multiple]").selectize({plugins: ["remove_button"]})',
+                ['block' => true]
+            );
+            ?>
```

Comment:
I actually have a similar thing for forms in my project, including normal selects. I wonder if moving this to local.js and apply to all selects would make more sense?